### DISCUSSION TITLE
The website can provide install Progressive Web Apps (PWA) way

### DISCUSSION THREAD

**Discussion by Zhengqbbb** [unocss/unocss#1349] [Ideas]
Hi guys

I've recently been obsessed with installing web pages as web apps, that **it's convenient to query the guide**, Would like if the unocss website can support it ~ ❤️

like icones: 
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/40693636/181509811-15c55318-6432-477b-a4c7-3adbc25f2131.png">
<img width="148" alt="image" src="https://user-images.githubusercontent.com/40693636/181509875-06e22d09-5fd2-449e-9d84-8660ba660f27.png">

**Comment by userquin** [comment]
@Zhengqbbb I'll check if we can add PWA support to the interactive docs, we need to do some test excluding some external urls... We also need to include offline support to prevent going to MSDN and remove some external link to regex sites.

/cc @antfu I'll check it if you want to add it
